packages/grafana-ui/src/components/ContextMenu/ContextMenu.tsx

```diff
@@ -188,7 +188,7 @@ export const ContextMenu: React.FC<ContextMenuProps> = React.memo(({ x, y, onClo
           renderItem={(item, index) => {
             return (
               <>
-                <ContextMenuGroup group={item} onClick={onClose} />
+                <ContextMenuGroupComponent group={item} onClick={onClose} />
```

Comment:
How does this rename relate to the packages bump?

Reply:
CI was complaining that `ContextMenuGroup` was being redeclared (the interface defined at the top of the file has the same name)

Reply:
Makes sense, thanks!